docs/examples/101/basic-publicip/main.json

```diff
@@ -87,7 +87,7 @@
   "outputs": {
     "ipFqdn": {
       "type": "string",
-      "value": "[reference(resourceId('Microsoft.Resources/deployments', 'publicIp'), '2019-10-01').outputs.ipFqdn.value]"
```

Comment:
>'2019-10-01' [](http://example.com/codeflow?start=86&length=12)

Is this change intentional?

Reply:
No, but it's actually more optimal codegen, so I left it in.

Reply:
I decided to revert this change to make the diffs easier to review. Now there are no modifications to existing files (apart from one with whitespace diffs), for greater confidence that the existing logic hasn't been changed.